### DISCUSSION TITLE
Stripe Integration asking for Google merchant ID

### DISCUSSION THREAD

**Discussion by virajsoni06** [swellstores/community#349] [Help] [answered]
Hi, the Stripe integration in Swell is asking for merchant ID whereas there is no merchant ID provided in Stripe for Google Pay. What's the solution here to use Google Pay?

Please note, we are using Swell's hosted checkout for payments

**Comment by SusannaSwell** [comment, marked as answer]
Hello @virajsoni06 , google merchant id should be received after sign-in at google merchant center https://www.google.com/retail/solutions/merchant-center/
it will be displayed at top right corner of the page, near your email
